### PR TITLE
Increase frontend list operation QPS to 30

### PR DESCRIPTION
**Pull request by feedmeapples**
**What changed?**
Increased QPS for frontend list workflows operation to 30 for both ES enabled and disabled environments

**Why?**
The default QPS was too low (3 and 1 respectively for ES enabled/disabled) and this would result in flaky failures of the list operation even at minimum load.
Burst rate limiting disallows QPS/10 requests made in a very short time. So increased the QPS to 30 to allow up to 3 almost simultaneous requests. 

**How did you test it?**
Tested the api by making 3-5 close to simultaneous requests from the Web UI 

**Potential risks**
These operations are expensive and actually making close to 30 requests per second might create a noticeable load. @wxing1292 Should fix the burst limiting to allow higher numbers so we can lower QPS to lower than 30

### CODE REVIEW

File: service/frontend/service.go [hunk 1]
```diff
@@ -136,9 +136,9 @@ func NewConfig(dc *dynamicconfig.Collection, numHistoryShards int32, enableReadF
 		PersistenceGlobalMaxQPS:                dc.GetIntProperty(dynamicconfig.FrontendPersistenceGlobalMaxQPS, 0),
 		VisibilityMaxPageSize:                  dc.GetIntPropertyFilteredByNamespace(dynamicconfig.FrontendVisibilityMaxPageSize, 1000),
 		EnableVisibilitySampling:               dc.GetBoolProperty(dynamicconfig.EnableVisibilitySampling, true),
-		VisibilityListMaxQPS:                   dc.GetIntPropertyFilteredByNamespace(dynamicconfig.FrontendVisibilityListMaxQPS, 1),
+		VisibilityListMaxQPS:                   dc.GetIntPropertyFilteredByNamespace(dynamicconfig.FrontendVisibilityListMaxQPS, 100),
```
Comment (wxing1292) [inline service/frontend/service.go:139]: i would at most default this to 10

File: service/frontend/service.go [hunk 2]
```diff
@@ -136,9 +136,9 @@ func NewConfig(dc *dynamicconfig.Collection, numHistoryShards int32, enableReadF
...
 		EnableReadVisibilityFromES:             dc.GetBoolPropertyFnWithNamespaceFilter(dynamicconfig.EnableReadVisibilityFromES, enableReadFromES),
-		ESVisibilityListMaxQPS:                 dc.GetIntPropertyFilteredByNamespace(dynamicconfig.FrontendESVisibilityListMaxQPS, 3),
+		ESVisibilityListMaxQPS:                 dc.GetIntPropertyFilteredByNamespace(dynamicconfig.FrontendESVisibilityListMaxQPS, 100),
```
Comment (wxing1292) [inline service/frontend/service.go:141]: i would at most default this to 30

File: service/frontend/service.go (hunk 1 above)
Comment (feedmeapples) [reply service/frontend/service.go]: changed

File: service/frontend/service.go (hunk 2 above)
Comment (feedmeapples) [reply service/frontend/service.go]: changed

**Review by alexshtin — APPROVED** [review]

**Review by wxing1292 — APPROVED** [review]

**Pull request merged**